open_seq2seq/data/speech2text/speech_utils.py

```diff
@@ -130,43 +142,42 @@ def fix_kv(text):
   return preprocessed_path
 
 
-def get_speech_features_from_file(filename,
-                                  num_features,
-                                  pad_to=8,
-                                  features_type='spectrogram',
-                                  window_size=20e-3,
-                                  window_stride=10e-3,
-                                  augmentation=None,
-                                  apply_window=False,
-                                  cache_features=False,
-                                  cache_format="hdf5",
-                                  cache_regenerate=False,
-                                  params={}):
+def get_speech_features_from_file(filename, params):
   """Function to get a numpy array of features, from an audio file.
       if params['cache_features']==True, try load preprocessed data from
       disk, or store after preprocesseng.
       else, perform preprocessing on-the-fly.
 
-Args:
-  filename (string): WAVE filename.
-  num_features (int): number of speech features in frequency domain.
-  features_type (string): 'mfcc' or 'spectrogram'.
-  window_size (float): size of analysis window in milli-seconds.
-  window_stride (float): stride of analysis window in milli-seconds.
-  augmentation (dict, optional): None or dictionary of augmentation parameters.
-      If not None, has to have 'time_stretch_ratio',
-      'noise_level_min', 'noise_level_max' fields, e.g.::
-        augmentation={
-          'time_stretch_ratio': 0.2,
-          'noise_level_min': -90,
-          'noise_level_max': -46,
-        }
-  apply_window (bool): whether to apply Hann window for mfcc and logfbank.
-        python_speech_features version should accept winfunc if it is True.
-Returns:
-  np.array: np.array of audio features with shape=[num_time_steps,
-  num_features].
-"""
+  Args:
+    filename (string): WAVE filename.
+    params (dict): the following parameters
+      num_features (int): number of speech features in frequency domain.
+      features_type (string): 'mfcc' or 'spectrogram'.
+      window_size (float): size of analysis window in milli-seconds.
+      window_stride (float): stride of analysis window in milli-seconds.
+      augmentation (dict, optional): None or dictionary of augmentation parameters.
+        If not None, has to have 'time_stretch_ratio',
+        'noise_level_min', 'noise_level_max' fields, e.g.::
+          augmentation={
+            'time_stretch_ratio': 0.2,
+            'noise_level_min': -90,
+            'noise_level_max': -46,
+          }
+      window_fn (bool): window function to apply, or None for no window
```

Comment:
should be window (string or None), not window_fn (bool)

Reply:
Fixed